lib/minitest/reporters/spec_reporter.rb

```diff
@@ -28,10 +38,20 @@ def report
 
       def record(test)
         super
-        test_name = test.name.gsub(/^test_: /, 'test:')
-        print pad_test(test_name)
-        print_colored_status(test)
-        print(" (%.2fs)" % test.time) unless test.time.nil?
+        if options[:show_order] == :before
+           print_colored_status(test) if options[:show_status]
+           test_name = test.name.gsub(/^test_(\d+_)?/, '  ')
```

Comment:
I'd expect that test_name will be extracted the same way in both cases.  Why do you use different regexp here?